server/eclipse-project/src/main/java/us/freeandfair/corla/json/CountyDashboardRefreshResponse.java

```diff
@@ -244,7 +251,8 @@ protected CountyDashboardRefreshResponse(final Long the_id,
     my_asm_state = the_asm_state;
     my_audit_board_asm_state = the_audit_board_asm_state;
     my_general_information = the_general_information;
-    my_audit_board = the_audit_board;
+    my_audit_board_count = the_audit_board_count;
```

Comment:
(As a general thought)
I'm keen on dropping the `my_` / `the_` style and `snake_case` in favor of idiomatic Java.

Reply:
I must have been in a "consistency over all" mood when I wrote this. We're in agreement I think, I prefer `this.auditBoardCount = auditBoardCount;` or something to that effect. It looked *particularly* ugly at this point in the code...

Reply:
Ah, gotcha. Let's leave the things that need to be serialized to JSON alone for now, then.